prototype/utils/testUtils.js

```diff
@@ -7,19 +7,24 @@ const projectName = require('../package.json').name;
 let sgt;
 
 module.exports = {
-  buildTransform(done) {
-    exec('sgtu-build-transform', (err) => {
+  build(done) {
+    exec('sgtu-build', (err) => {
```

Comment:
Did you have a reason for combining these? If this is called at the beginning of the transform test and at the beginning of the toUrl test, each one will be built twice when you run 'test'.

Reply:
also, can you modify this to return stdout and stderr and log those? That should fix the problem we were talking about yesterday where it fails silently.

Reply:
Do you think we should have `transform test` and `toUrl test` call its own build function?

Reply:
Yes, I think that would be better. Not a huge deal, but it should make the build run faster.

Reply:
If you don't have time to change it now we can merge it as-is and do this later.